JUnit portion of testsuite fails to compile on ppc64le

The current version of testsuite no longer compiles on PPC64LE.  I am seeing the following error:

```
$ make
[  0%] Built target testB_static
[  0%] Built target testB
[  0%] Built target testA_static
[  0%] Building CXX object CMakeFiles/testSuite.dir/src/JUnitOutputDriver.cpp.o
/g/g0/martymcf/devel/src/testsuite_9.2/src/JUnitOutputDriver.cpp: In constructor 'JUnitOutputDriver::JUnitOutputDriver(void*)':
/g/g0/martymcf/devel/src/testsuite_9.2/src/JUnitOutputDriver.cpp:13:50: error: 'getpid' was not declared in this scope
     results_log_name << "test_results" << getpid() << ".xml";
                                                  ^
make[2]: *** [CMakeFiles/testSuite.dir/src/JUnitOutputDriver.cpp.o] Error 1
make[1]: *** [CMakeFiles/testSuite.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/JUnitOutputDriver.cpp
+++ b/src/JUnitOutputDriver.cpp
@@ -4,6 +4,11 @@
 
 #include "JUnitOutputDriver.h"
 
+#if !defined(os_windows_test)
+#include <sys/types.h>
+#include <unistd.h>
+#endif
+
 JUnitOutputDriver::JUnitOutputDriver(void *data) : StdOutputDriver(data),
                                                    group_failures(0),
                                                    group_skips(0),

